tools/generate-sponsors.js

```diff
@@ -26,20 +26,25 @@ const got = require("got");
 
 const SPONSORS_URL =
 	"https://raw.githubusercontent.com/eslint/eslint.org/main/src/_data/sponsors.json";
-const NEW_FILE_PATH = "../src/_data/sponsors.md";
+const NEW_FILE_PATH = "./sponsors.md";
```

Comment:
Let's create a new folder for this, it doesn't belong in the `tools` directory. Maybe `includes`?